package.json

```diff
@@ -24,19 +24,19 @@
   ],
   "scripts": {
     "start": "webpack-dev-server --config webpack/dev.js",
-    "build": "npm run clean && npm run build:lib && npm run build:es6 && mkdir umd && npm run build:umd",
+    "clean": "rimraf lib es6 umd",
+    "build": "npm-run-all -s clean -p build:** -s size",
     "build:lib": "babel src -d lib --presets es2015,stage-0,react",
     "build:es6": "babel src -d es6",
-    "build:umd": "webpack --config webpack/umd.js && npm run compress:umd",
+    "build:umd": "mkdirp umd && webpack --config webpack/umd.js",
```

Comment:
using `mkdirp ` instead of `mkdir` for windows compatibility 

also `umd` dir needs to be only created for `umd` build, thus co-located the commands.